src/core.ts

```diff
@@ -0,0 +1,212 @@
+import Config from './config';
+import Logger from './logger';
+import Storage from './storage';
+import Navigator from './navigator';
+import { RadarPublishableKeyError } from './errors';
+
+import AddressesAPI from './api/addresses';
+import ConfigAPI from './api/config';
+import ContextAPI from './api/context';
+import ConversionsAPI from './api/conversions';
+import GeocodingAPI from './api/geocoding';
+import RoutingAPI from './api/routing';
+import SearchAPI from './api/search';
+import TrackAPI from './api/track';
+import TripsAPI from './api/trips';
+import VerifyAPI from './api/verify';
+
+import SDK_VERSION from './version';
+
+import type {
+  Location,
+  RadarAutocompleteParams,
+  RadarConversionParams,
+  RadarDistanceParams,
+  RadarForwardGeocodeParams,
+  RadarMatrixParams,
+  RadarMetadata,
+  RadarOptions,
+  RadarReverseGeocodeParams,
+  RadarSearchGeofencesParams,
+  RadarSearchPlacesParams,
+  RadarTrackParams,
+  RadarTripOptions,
+  RadarValidateAddressParams,
+} from './types';
+
+const isSecretKey = (key: string): boolean => (
+  key.includes('_sk_')
+);
+const isLiveKey = (key: string): boolean => (
+  key.includes('_live_')
+);
+
+class Radar {
```

Comment:
in `index.ts`, should we do 

```
class Radar extends RadarCore {
  // "ui" namespace
  public static get ui() {
    return {
      maplibregl: MapUI.getMapLibre(),
      map: MapUI.createMap,
      marker: MapUI.createMarker,
      autocomplete: AutocompleteUI.createAutocomplete,
    };
  }
}
```

to minimize code duplication

Reply:
Nice